### PR TITLE
Dateselector and usage in EventCreationScreen

### PR DESCRIPTION
**Pull request by Athanaze**

### CODE REVIEW

File: app/src/main/java/com/monkeyteam/chimpagne/ui/EventCreationScreen.kt
Comment (Kuriv2001) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/EventCreationScreen.kt:1]: Change print to log

File: app/src/main/java/com/monkeyteam/chimpagne/ui/components/DateSelector.kt
Comment (Kuriv2001) [inline app/src/main/java/com/monkeyteam/chimpagne/ui/components/DateSelector.kt:1]: Some feedback should be added, when date has been selected, to see if the right date has been added.

**Review by Kuriv2001 — CHANGES_REQUESTED** [review]
The code looks really concise and well structured, good job! It would be good to add some feedback to the date selection, so the user knows that the right date has been selected.

**Comment by juaniac** [comment]
I believe the changes made here are already done in the main branch, so this branch will be discontinued. Eventually you can create a new branch to make updates to the changes on the DateSelector on the main branch.

**Pull request closed without merging**